script/transform/page__alto

```diff
@@ -0,0 +1 @@
+alto__page
```

Comment:
How should the conversion from PAGE to ALTO work when it uses the same call as ALTO to PAGE?

Reply:
The alto__page command will call a java library. The help there says:
```
$ java -jar /usr/local/share/ocr-fileformat/vendor/JPageConverter/PageConverter.jar
PAGE Converter

PRImA Research Lab, University of Salford, UK

Arguments:

  -source-xml <XML file>        PAGE XML file to convert.
     OR
  -source-json <JSON file>      JSON file to convert (e.g. Google Cloud Vision output).

  -target-xml <XML file>        Output PAGE XML file.

  -convert-to <schema version>  Target PAGE schema version. (optional)
              Available versions:
                 LATEST
                 2018-07-15
                 2017-07-15
                 2016-07-15
                 2013-07-15
                 2010-03-19
                 ALTO

  -set-gtsid <ID|prefix[start,end]>   To set the the GtsId field. (optional)
         Usage:
              Use <ID> to set a specific GtsId.
              Use <prefix[start,end]> to extract the GtsId from the
              filename of -source-xml.
         Examples:
              -set-gtsid 00236178     Given ID
              -set-gtsid pc-[0,7]     Prefix + first 8 characters of filename

  -text-filter <XML file>   Applies filter to the text content. (optional)
         For instructions on how to define filter rules see the user guide.

  -neg-coords <mode>   Handle negative coordinates (optional)
       Modes:
         removeObj - If an object contains one or more points with negative
                     coordinates, remove the whole object.
         toZero    - Change negative values to 0
```
It seems that the source xml can also be a ALTO file and this can be easily been detected by the script. However, to control the output we need IMO the parameter `-convert-to` which I also used in the PR #105.